README.md

```diff
@@ -298,6 +181,23 @@ resp, err := client.Search(indexUID).Search(meilisearch.SearchRequest{
 }
 ```
 
+## 🤖 Compatibility with MeiliSearch
+
+This package only guarantees the compatibility with the [version v0.15.0 of MeiliSearch](https://github.com/meilisearch/MeiliSearch/releases/tag/v0.15.0).
+
+## 📖 Documentation and Examples
+
+MeiliSearch documentation provides **examples** and a detailed explanation of every one of its features and functionalities, including examples on how to implement them **using this SDK**.
+
+Please read the [guides available in the documentation](https://docs.meilisearch.com/guides/) or check the [API references](https://docs.meilisearch.com/references/) to find the one that you need!
+
+The following sections may interest you:
+
+- [Manipulate documents](https://docs.meilisearch.com/references/documents.html)
+- [Search](https://docs.meilisearch.com/references/search.html)
+- [Manage the indexes](https://docs.meilisearch.com/references/indexes.html)
+- [Configure the index settings](https://docs.meilisearch.com/references/settings.html)
```

Comment:
```suggestion
- **Manipulate documents**: see the [API reference](https://docs.meilisearch.com/references/documents.html) or follow our [guide](https://docs.meilisearch.com/guides/main_concepts/documents.html)
- **Search**: see the [API reference](https://docs.meilisearch.com/references/search.html) opr follow our [guide](https://docs.meilisearch.com/guides/main_concepts/search.html)
- **Manage the indexes**: see the [API reference](https://docs.meilisearch.com/references/indexes.html) or follow our [guide](https://docs.meilisearch.com/guides/main_concepts/indexes.html)
- **Configure the index settings**: see the [API reference](https://docs.meilisearch.com/references/settings.html) our [guide on relevancy](https://docs.meilisearch.com/guides/main_concepts/relevancy.html) 
```

Reply:
Despite some typos, I like this idea, we will discuss it in meetings tomorrow 🙂 (I know Charlotte wants to talk about this section)

Reply:
I think I corrected all the typos ;)
Let's discuss it then!

Reply:
API reference -> API references